src/Cache/CacheClient.php

```diff
@@ -832,19 +1313,21 @@ public function dictionarySetFields(string $cacheName, string $dictionaryName, a
      * * DictionaryGetFieldsMiss<br>
      * * DictionaryGetFieldsError<br>
      * Pattern matching can be to operate on the appropriate subtype:<br>
-     * <code>if ($hit = $response->asHit()) {<br>
-     * &nbsp;&nbsp;// get a list of responses corresponding to the list of requested values<br>
-     * &nbsp;&nbsp;// each item in the list is an instance of one of the following:<br>
-     * &nbsp;&nbsp;// - DictionaryGetFieldHit<br>
-     * &nbsp;&nbsp;// - DictionaryGetFieldMiss<br>
-     * &nbsp;&nbsp;// - DictionaryGetFieldError<br>
-     * &nbsp;&nbsp;$responseTypes = $hit->responses();
-     * &nbsp;&nbsp;// get a dictionary of responses mapping requested field name keys to their values<br>
-     * &nbsp;&nbsp;// fields that were not found in the cache dictionary are omitted<br>
-     * &nbsp;&nbsp;$valuesDict = $hit->valuesDictionary();
-     * } elseif ($error = $response->asError()) {<br>
-     * &nbsp;&nbsp;// handle error condition<br>
-     * }</code>
+     * <code>
+     * if ($hit = $response->asHit()) {
```

Comment:
lol i never noticed before how easy it is to misread `$hit` if you are scrolling through a PR pretty quickly :)